pkg/sdk/go/channels_test.go

```diff
@@ -15,7 +15,10 @@ import (
 )
 
 var (
-	channel      = sdk.Channel{ID: "001", Name: "test"}
+	ch1ExtID     = sdk.Channel{ID: "fe6b4e92-cc98-425e-b0aa-000000000001", Name: "test1"}
+	ch2ExtID     = sdk.Channel{ID: "fe6b4e92-cc98-425e-b0aa-000000000002", Name: "test2"}
+	channel1     = sdk.Channel{Name: "test1"}
```

Comment:
I'd use `ch1` to be consistent with other namings

Reply:
I am moving them to ch1, ch2, ch3 now.